coordinator/test/mock_prover.go

```diff
@@ -110,23 +110,23 @@ func (r *mockProver) login(t *testing.T, challengeString string) string {
 }
 
 func (r *mockProver) healthCheckSuccess(t *testing.T) bool {
-	var result types.Response
+	var result ctypes.Response
 	client := resty.New()
 	resp, err := client.R().
 		SetResult(&result).
-		Get("http://" + r.coordinatorURL + "/coordinator/v1/health")
+		Get("http://" + r.coordinatorURL + "/coordinator/v1/challenge")
 	assert.NoError(t, err)
 	assert.Equal(t, http.StatusOK, resp.StatusCode())
 	assert.Equal(t, ctypes.Success, result.ErrCode)
 	return true
 }
 
 func (r *mockProver) healthCheckFailure(t *testing.T) bool {
-	var result types.Response
+	var result ctypes.Response
 	client := resty.New()
 	resp, err := client.R().
 		SetResult(&result).
-		Get("http://" + r.coordinatorURL + "/coordinator/v1/health")
+		Get("http://" + r.coordinatorURL + "/coordinator/v1/challenge")
```

Comment:
not sure why we need to change this?

Reply:
Because I move the health to another observability Server,  In this unit test only start a coordinator server.